### PR TITLE
Include custom image sizes in shortcode UI

### PR DESCRIPTION
**Pull request by goldenapples**
Runs the image sizes available through the `image_size_names_choose`
filter, so that custom image sizes added in themes can also be available
for selection through the UI here.

Addresses #24.

### CODE REVIEW

File: inc/class-img-shortcode.php [hunk 1]
```diff
@@ -16,6 +16,30 @@ class Img_Shortcode {
...
+			'thumbnail' => __( 'Thumbnail', 'image-shortcake' ),
+			'medium'    => __( 'Medium',    'image-shortcake' ),
+			'large'     => __( 'Large',     'image-shortcake' ),
+			'full'      => __( 'Full size', 'image-shortcake' ),
+		);
+		$sizes_available = array();
+
+		foreach ( apply_filters( 'image_size_names_choose', $default_sizes ) as $key => $name ) {
+
+			$size = get_option( "${key}_size_w" );
```
Comment (danielbachhuber) [inline inc/class-img-shortcode.php]: Shouldn't this syntax be `{$key}`?

File: inc/class-img-shortcode.php [hunk 2]
```diff
@@ -16,6 +16,30 @@ class Img_Shortcode {
 	 * filter.
 	 */
 	public static function get_shortcode_ui_args() {
+		global $_wp_additional_image_sizes;
+
+		$default_sizes = array(
+			'thumbnail' => __( 'Thumbnail', 'image-shortcake' ),
+			'medium'    => __( 'Medium',    'image-shortcake' ),
+			'large'     => __( 'Large',     'image-shortcake' ),
+			'full'      => __( 'Full size', 'image-shortcake' ),
```
Comment (danielbachhuber) [inline inc/class-img-shortcode.php]: These should be `esc_html__()`, otherwise VIP will give us pushback.

**Comment by danielbachhuber** [comment]
Couple minor issues to address.

File: inc/class-img-shortcode.php (hunk 1 above)
Comment (goldenapples) [inline inc/class-img-shortcode.php]: Ugh, yup. This isn't bash.

Comment (goldenapples) [inline inc/class-img-shortcode.php]: But it does work, apparently: http://php.net/manual/en/language.types.string.php#language.types.string.syntax.double

File: inc/class-img-shortcode.php (hunk 2 above)
Comment (goldenapples) [inline inc/class-img-shortcode.php]: Its being escaped below in line 41, after adding the width to it. Didn't want to double-escape it. Should it be marked or commented somehow to pass automated sniffs?

File: inc/class-img-shortcode.php (hunk 1 above)
Comment (goldenapples) [inline inc/class-img-shortcode.php]: Changed in c909811.

File: inc/class-img-shortcode.php (hunk 2 above)
Comment (goldenapples) [inline inc/class-img-shortcode.php]: Escaped in c909811.

**Comment by goldenapples** [comment]
Updated to respond to comments. Can you take another look at it?

**Pull request merged**